test/prism_regression/literal_integer.rb

```diff
@@ -1,9 +1,50 @@
 # typed: false
 
-42
+# Comments represent the value in the parse tree
```

Comment:
This is why I liked to use incrementing numbers (or other unique values), so it's easier to associate values between the two files.

I fear these comments might end up changing or end up with mistakes, but it's probably fine